lib/storySchema.ts

```diff
@@ -0,0 +1,44 @@
+// This validation schema is shared between client and server
+// Used in: `/components/form`, '/pages/api/stories/index.ts'
+
+import * as yup from 'yup'
+
+export const TITLE_CHAR_LIMIT = 75
+export const STORY_WORD_LIMIT = 1000
+
+const schema = yup.object().shape({
+  title: yup
```

Comment:
We share this schema between client and server which is nice.

Reply:
This is really cool.